enhancements/console/dynamic-plugins.md

```diff
@@ -363,6 +424,8 @@ console will only load the correct plugin.
 ## Implementation History
 
 * 2020-07-24 - Initial [dynamic plugin PR](https://github.com/openshift/console/pull/6101) opened.
+* 2020-11-26 - Allow static plugins to use new extension types with code references
```

Comment:
Listed openshift/console#7163 here since it marks an important milestone in the transition process from static to dynamic plugins.